LayoutTests/imported/w3c/web-platform-tests/trusted-types/WorkerGlobalScope-importScripts-expected.txt

```diff
@@ -7,14 +7,6 @@ PASS importScripts with two URLs, both strings, in dedicated worker
 PASS importScripts with two URLs, one trusted, in dedicated worker
```

Comment:
This test is a flaky failure but I would be surprised if these changes are needed? Can probably revert